src/main/java/blackjack/controller/BlackjackController.java

```diff
@@ -28,85 +29,98 @@ public BlackjackController(InputView inputView, OutputView outputView) {
+        return new Dealer(new RandomCardShuffleStrategy());
     }
 
-    private CardDeck generateCardDeck() {
-        return Arrays.stream(Emblem.values())
-                .map(CardFactory::create)
-                .flatMap(Collection::stream)
-                .collect(collectingAndThen(toList(), CardDeck::new));
+    private Players generatePlayers() {
+        String readPlayerNames = replyOnException(inputView::readPlayerNames);
+        List<String> playerNames = Converter.stringToList(readPlayerNames);
+
+        return playerNames.stream()
+                .map(this::generatePlayer)
+                .collect(collectingAndThen(toList(), Players::new));
     }
 
-    private Players generatePlayers() {
-        List<String> playerNames = inputView.readPlayerNames();
-        return new Players(playerNames);
+    private Player generatePlayer(String playerName) {
+        BattingAmount battingAmount = replyOnException(() -> readBattingAmount(playerName));
+        return new Player(playerName, battingAmount);
+    }
+
+    private BattingAmount readBattingAmount(String playerName) {
+        int readBattingAmount = inputView.readBattingAmount(playerName);
+        return new BattingAmount(readBattingAmount);
+    }
+
+    private void playBlackjackGame(BlackjackGame blackjackGame, Dealer dealer, Players players) {
+        blackjackGame.start();
+        showCardsAfterFirstDeal(dealer, players);
+
+        dealToParticipants(players, dealer, blackjackGame);
     }
 
-    private GameResult playBlackjackGame(Dealer dealer, Players players, BlackjackGame blackjackGame) {
-        blackjackGame.startGame();
-        outputView.printCardHand(dealer, players);
+    private void showCardsAfterFirstDeal(Dealer dealer, Players players) {
+        ParticipantDto dealerDto = ParticipantDto.from(dealer);
+        ParticipantsDto playersDto = ParticipantsDto.toDtoWithoutDealer(players);
 
-        dealToParticipants(dealer, players, blackjackGame);
-        return blackjackGame.createGameResult();
+        outputView.printCardsAfterFirstDeal(dealerDto, playersDto);
     }
 
-    private void dealToParticipants(Dealer dealer, Players players, BlackjackGame blackjackGame) {
+    private void dealToParticipants(Players players, Dealer dealer, BlackjackGame blackjackGame) {
         dealToPlayers(players, blackjackGame);
-        dealToDealer(blackjackGame);
-        outputView.printCardHandWithScore(dealer, players);
+        dealToDealer(dealer, blackjackGame);
     }
 
     private void dealToPlayers(Players players, BlackjackGame blackjackGame) {
-        for (int i = 0; i < players.count(); i++) {
-            Player player = players.findPlayerByIndex(i);
-            dealToPlayer(blackjackGame, i, player);
+        for (int playerIndex = 0; playerIndex < players.count(); playerIndex++) {
+            Player player = players.findPlayerByIndex(playerIndex);
+            dealToPlayer(player, blackjackGame);
```

Comment:
Stream 을 활용해보면 어떨까요? 😀

```suggestion
      players
      .getAllPlayers()
      .forEach(player -> dealToPlayer(player, blackjackGame));
```